docs/01_cva6_user/Parameters_Configuration.rst

```diff
@@ -0,0 +1,67 @@
+﻿..
+   Copyright (c) 2023 OpenHW Group
+   Copyright (c) 2023 Thales DIS design services SAS
+
+   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+.. Level 1
+   =======
+
+   Level 2
+   -------
+
+   Level 3
+   ~~~~~~~
+
+   Level 4
+   ^^^^^^^
+
+.. _cva6_parameters_configuration:
+
+Parameters and Configuration
+============================
+
+Parameters
+----------
+We start with a table of parameters as they have an impact on almost all subsequent sections.
+Suggestion to use the SystemVerilog names of the parameters (instead of another convention) as a reference. We need to make a link between parameters and their impact on the supported extensions.
+Jean-Roch said he has something.
+
+Configurations
+--------------
+A configuration is a fixed set of parameters.
+We list the parameters of the configuration for which verification activities have started.
+Give step 1 configuration (Jean-Roch?)
+

```

Comment:
We should have a file break here (as in https://docs.google.com/document/d/1W3gBZPk0tNWquXo5CkcLWU07CdxaaT0gzsnhQ_dpiIo/edit?usp=sharing) and the next headers should be promoted accordingly. Not blocking for this PR.